UITextView+Placeholder/UITextView+Placeholder.m

```diff
@@ -55,14 +55,7 @@ - (void)swizzledDealloc {
 #pragma mark `defaultPlaceholderColor`
 
 + (UIColor *)defaultPlaceholderColor {
-    static UIColor *color = nil;
-    static dispatch_once_t onceToken;
-    dispatch_once(&onceToken, ^{
-        UITextField *textField = [[UITextField alloc] init];
-        textField.placeholder = @" ";
-        color = [textField valueForKeyPath:@"_placeholderLabel.textColor"];
-    });
-    return color;
+    return [UIColor colorWithWhite:0.70f alpha:1.0f];
```

Comment:
**Reason 1.**
This method is allowed by Apple. It's just private property, not a private API. All of my applications using this are uploaded to AppStore.

or you can refer this StackOverflow answers: http://stackoverflow.com/questions/1340224/iphone-uitextfield-change-placeholder-text-color

**Reason 2**
Apple is lying. Placeholder label color isn't 70% grey.

```swift
let textField = UITextField()
textField.placeholder = "A"
if let color = textField.valueForKeyPath("_placeholderLabel.textColor") as? UIColor {
    println(color) // UIDeviceRGBColorSpace 0 0 0.0980392 0.22
}
```

It's 22% of very dark blue.